Formula/xclogparser.rb

```diff
@@ -0,0 +1,18 @@
+class Xclogparser < Formula
+  desc "Tool to parse the SLF serialization format used by Xcode"
+  homepage "https://github.com/spotify/XCLogParser"
+  url "https://github.com/spotify/XCLogParser/archive/0.2.8.tar.gz"
+  sha256 "d8368a561c8e3d4a7ce049feaa5f6bc6fbda532c8c0cc8bc44feab61c9aab0f9"
+
+  depends_on :xcode => :build
+  depends_on :macos => :mojave
```

Comment:
If it doesn't build on all macOS versions it is better to maintain it in a personal tap.

Reply:
Attempting to remove this requirement, How can I avoid bottling for systems lower than Mojave?

Reply:
Why would you want to avoid that?